src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/Node.java

```diff
@@ -187,4 +194,20 @@ public void setLocalFlowUnit(T localFlowUnit) {
   public void readRcaConf(RcaConf conf) {
     return;
   }
+
+  public void setAppContext(final AppContext appContext) {
```

Comment:
add javadoc for this function ?

Reply:
Do we really want a javadoc for a setter ?